iphone/Maps/Core/Location/MWMLocationManager.mm

```diff
@@ -312,6 +358,7 @@ + (void)setMyPositionMode:(MWMMyPositionMode)mode
     case MWMRouterTypePublicTransport:
     case MWMRouterTypePedestrian: manager.geoMode = GeoMode::PedestrianRouting; break;
     case MWMRouterTypeBicycle: manager.geoMode = GeoMode::BicycleRouting; break;
+    case MWMRouterTypeRuler: break;
```

Comment:
Can you please check where `MWMRouter type` is used in other places, to make sure that MWMRouterTypeRuler is handled properly everywhere? It was added way later after other router types.

Reply:
At first glance MWMRouterTypeRuler is set up correctly everywhere.
Maybe we should use here some other geomode? for example - InPosition ot pending - it will trigger the `CLActivityTypeOther`
```objc
  switch (geoMode)
  {
  case GeoMode::Pending:
  case GeoMode::InPosition:
  case GeoMode::NotInPosition:
  case GeoMode::FollowAndRotate:
    locationManager.activityType = CLActivityTypeOther;
    break;
  case GeoMode::VehicleRouting:
    locationManager.activityType = CLActivityTypeAutomotiveNavigation;
    break;
  case GeoMode::PedestrianRouting:
  case GeoMode::BicycleRouting:
    locationManager.activityType = CLActivityTypeOtherNavigation;
    break;
  }
```

Reply:
When navigation over an imported track or over hand-drawn ruler track will be implemented, it should be equal to CLActivityTypeOtherNavigation